board/batocera/amlogic/odroidn2/post-image-script.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -e
+
+# the aim of this script is to prepare output/batocera/boot depending on your target
+# HOST_DIR = host dir
+# BOARD_DIR = board specific dir
+# BUILD_DIR = base dir/build
+# BINARIES_DIR = images dir
+# TARGET_DIR = target dir
+# BATOCERA_BINARIES_DIR = batocera binaries
+
+HOST_DIR=$1
+BOARD_DIR=$2
+BUILD_DIR=$3
+BINARIES_DIR=$4
+TARGET_DIR=$5
+BATOCERA_BINARIES_DIR=$6
+
+MKIMAGE=${HOST_DIR}/bin/mkimage
+
+mkdir -p "${BATOCERA_BINARIES_DIR}/boot/boot"     || exit 1
+mkdir -p "${BATOCERA_BINARIES_DIR}/boot/extlinux" || exit 1
+"${HOST_DIR}/bin/mkimage" -A arm64 -O linux -T kernel -C none -a 0x1080000 -e 0x1080000 -n linux -d "${BINARIES_DIR}/Image" "${BATOCERA_BINARIES_DIR}/boot/boot/linux" || exit 1
+cp "${BOARD_DIR}/boot/boot-logo.bmp.gz"            "${BATOCERA_BINARIES_DIR}/boot"                                    || exit 1
```

Comment:
1. With `set -e` there is no need for `|| exit 1`
2. Without `set -e` it should be just `|| exit`, `exit` returns the value to the left of the `||`.